llm/qwen/gui.yaml

```diff
@@ -13,7 +13,8 @@
 # you can click on it to open the GUI.
```

Comment:
We can also consider switching to vllm's GUI server, as that has simpler prompt templates for different models, but not feeling strongly.
https://github.com/skypilot-org/skypilot/blob/master/llm/llama-3/gui.yaml

Reply:
great suggestion! setup after switching to vllm GUI was much faster than fastchat since it doesn't need torch and other dependencies. updated now.